lua/orgmode/colors/from_config/init.lua

```diff
@@ -0,0 +1,58 @@
+local ts = require('orgmode.treesitter.compat')
+local tree_utils = require('orgmode.utils.treesitter')
+local utils = require('orgmode.utils')
+
+local M = {}
+
+---@class OrgConfigHighlighter
+---@field get_queries function()
```

Comment:
_todo_: Need to figure out how to define the right return type for the `function` field here.